Makefile

```diff
@@ -49,6 +50,9 @@ test_redis_go: compile download_starter_testers
 
 test_redis_c: compile download_starter_testers
 	bundle exec ruby tests/test_all.rb redis c
+	
+test_redis_rust: compile download_starter_testers
+	bundle exec ruby tests/test_all.rb redis rust
```

Comment:
What happens when you run `make test_redis_rust`, @syed-ghufran-hassan? Do you see tests being run?

Reply:
When I ran this, here is the output I got: 

```
sudo rm -rf compiled_starters/*
Password:
bundle exec ruby template_compiler/compile.rb
mkdir -p .starter_testers
test -f .starter_testers/redis || make download_redis_starter_tester
test -f .starter_testers/docker || make download_docker_starter_tester
test -f .starter_testers/git || make download_git_starter_tester
bundle exec ruby tests/test_all.rb redis rust
Found 23 tests
Filtering by course:redis
Filtering by language:rust
No tests found!
make: *** [test_redis_rust] Error 1
```

It says no tests found. I think you'll want to look into this a bit more - maybe try looking at how the tests work? Try running `make test_redis_ruby` locally, and you'll see what I mean.

Reply:
No sir but I am checking it as I was unable to run the test due to windows OS

Reply:
Ah, okay. I _think_ the only dependencies you'll need are Make, Ruby and Docker. To run Docker on windows, you'll probably need to use [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10), or install [Docker for Windows](https://docs.docker.com/docker-for-windows/install/).

Reply:
yes sir I have installed docker for windows now I am checking if it is running okay